src/server/repository/assessment/assessment/getCreateSchemaDDL.ts

```diff
@@ -306,7 +306,10 @@ export const getCreateSchemaCycleOriginalDataPointViewDDL = (assessmentCycleSche
                               coalesce(rv.other_planted_forest_area)
                      else null
                      end as total
-          from raw_values rv)
+          from raw_values rv),
+      total_land_area as (
+        select * from ext_data.node where variable_name = 'totalLandArea'
```

Comment:
maybe include also table_name in the where clause ? just to be 100% sure. variable names are unique per table.